prow/hook/server.go

```diff
@@ -42,6 +42,11 @@ type Server struct {
 func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	defer r.Body.Close()
 
+	// Our health check uses GET, so just kick back a 200.
+	if r.Method == http.MethodGet {
```

Comment:
Can we specify that r.URL.Path == "/health" too?

Reply:
The "/hook" endpoint has special routing. I'd prefer to hit exactly this URL, so I don't need to worry about consistency.